main.py

```diff
@@ -424,6 +424,8 @@ def emailLogForCourseID(courseID, recipients):
 
     try:
         server = smtplib.SMTP(config.Application.Email.SMTP_SERVER)
+        logger.debug("mail server: "+config.Application.Email.SMTP_SERVER)
+        server.set_debuglevel(True)
```

Comment:
Should this always be in debug mode?

Reply:
Nope, this will be settable in the future.